parsing-utilities/src/main/java/gov/nasa/jpl/aerie/json/package-info.java

```diff
@@ -0,0 +1,181 @@
+/**
+ * A toolkit for authoring bidirectional JSON parsers with parser combinators.
+ *
+ * <p> This library provides the basic building blocks for defining serialization formats for any Java type,
+ * together with "parsers" which can convert both to and from these formats. A format is defined grammatically, by
+ * composing simpler pre-existing formats into more complex assemblies with the help of general-purpose combiners.
+ * We prioritize ease-of-use and expressivity over efficiency: parsers defined with this library may be slower than
+ * those defined with other libraries, but our goal is for them to be more maintainable in certain dimensions
+ * than alternatives. </p>
+ *
+ * <p> We take the position that serialization is a separate and orthogonal concern from domain modeling. Domain types
+ * ought not favor a particular mode of serialization, as business needs can easily change over time. Formats can be
+ * authored using this library without any need to modify the domain types they operate on. Moreover, where other
+ * libraries may infer details about the serialization format by inspecting the class definitions of domain types, we
+ * accept some potential overhead in repeating ourselves to cleanly separate the concern of what <i>developers</i> call
+ * a field from what <i>clients</i> call it -- and whether their data is structually organized the same way at all. </p>
+ *
+ * <p> We <b>eschew reflection and convention</b> in favor of explicit control over how Java values are modeled in JSON.
+ * Other libraries may require little to no configuration by relying on reflection and convention, but needs outside
+ * the happy path begin to look very different (and ad-hoc) from where you started. This library tries to provide a
+ * uniform and consistent authoring experienc: if you can author a simple format, odds are you know everything you need
```

Comment:
experienc -> experience

Reply:
Thanks; fixed. (Also fixed the &lt;/li&gt; that should have been a &lt;/p&gt;.)